setup.py

```diff
@@ -13,7 +13,7 @@
 
 
 install_requires = (
-    'Django>=1.11,<3.1',
+    'Django>=1.11,<3.2',
```

Comment:
```suggestion
    'Django>=1.11,<3.3',
```
I was about to put up a PR for a Django 3.2.12 upgrade my team is working on, but found your PR here already close to what we needed to create. Thoughts on this minor version bump?